api/version.go

```diff
@@ -0,0 +1,130 @@
+package api
```

Comment:
Lifted this from https://github.com/buildpacks/pack/blob/master/internal/api/version.go. If this is the right way to go we could remove pack's version and simply rely on this one.